Magical Nights/monsters/golems.json

```
@@ -215,5 +215,71 @@
     "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "HEARS", "NO_BREATHE", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_orchigolem",
+    "type": "MONSTER",
+    "name": "orichalcum golem",
+    "description": "A large, humanoid golem made from orichalcum. It's arms are fashioned into sharp blades.",
+    "default_faction": "robot",
+    "bodytype": "human",
+    "species": [ "ROBOT" ],
+    "diff": 2,
+    "volume": "120 L",
+    "weight": "600 kg",
+    "hp": 600,
+    "speed": 70,
+    "material": [ "orichalcum_metal" ],
+    "symbol": "X",
+    "color": "light_green",
+    "aggression": 9,
+    "morale": 100,
+    "melee_skill": 10,
+    "melee_dice": 5,
+    "melee_dice_sides": 12,
+    "melee_cut": 10,
+    "armor_bash": 40,
+    "armor_cut": 40,
+    "armor_bullet": 35,
+    "armor_fire": 40,
+    "dodge": 0,
+    "vision_day": 40,
+    "vision_night": 40,
+    "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
+    "death_function": [ "BROKEN" ],
+    "flags": [ "SEES", "NO_BREATHE", "ACIDPROOF", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_mithrgolem",
+    "type": "MONSTER",
+    "name": "mithril golem",
+    "description": "A large, humanoid golem made from mithril. This golem is practically invulnerable. Luckily, it is less aggressive",
+    "default_faction": "robot",
+    "bodytype": "human",
+    "species": [ "ROBOT" ],
+    "diff": 2,
+    "volume": "120 L",
+    "weight": "945 kg",
+    "hp": 800,
+    "speed": 80,
+    "material": [ "mithril_metal" ],
+    "symbol": "X",
+    "color": "light_blue",
```

```suggestion
    "color": "light_blue",
    "looks_like": "mon_irongolem",
```
Nice little (sensible, imo) fallback regarding the appearance of the golem.